### PR TITLE
Adding Scott for ToC 2021 Election

### PR DESCRIPTION
**Pull request by n3wscott**

### CODE REVIEW

**Comment by omerbensaadon** [comment]
+1 seems long overdue 💯

**Comment by zhongduo** [comment]
+1 Scott is very impactful to the project.

**Comment by travis-minke-sap** [comment]
+1 Scott is awesome!

**Comment by retgits** [comment]
+1!

**Comment by abrennan89** [comment]
+1 🎉

**Comment by csantanapr** [comment]
+1 Scott knows all parts of the project very well

**Comment by itsmurugappan** [comment]
+1

**Comment by lance** [comment]
I can't really speak for Scott's regular Knative contributions, but he has been a steady voice and prolific contributor to the CNCF CloudEvents project, in which I am more actively involved than Knative. If his contributions there reflect in any way on his dedication to and thoughtfulness towards Knative, I'm a strong +1.

**Comment by embano1** [comment]
Scott continuous to have a huge impact on the project, especially on the eventing side with his work and contributions on critical areas such as conformance testing, schema and API validation and accessibility with the tools he created around Knative, eg Sockeye. His chopping wood and carrying water mentality is important for the growing adoption of Knative, especially in the enterprise. 

He‘s also been very impactful on the community, bringing in new talent, lowering the barriers of entry, and constantly thriving for the best user experience of the project. 

IMHO a talented and unique allrounder in the amazing Knative project.

**Comment by xtreme-sameer-vohra** [comment]
+1

File: elections/2021-TOC/candidate-n3wscott.md [hunk 1]
```diff
@@ -0,0 +1,23 @@
+---
+
+name: Scott Nichols ID: n3wscott info:
+
```
Comment (dprotaso) [inline elections/2021-TOC/candidate-n3wscott.md:4]:
```suggestion
name: Scott Nichols 
ID: n3wscott 
info:
```

**Comment by danpopSD** [comment]
VERY BIG +1  -  easily one of the best engineers i have had the privilege to collaborate with.

**Review by vaikas — APPROVED** [review]
/lgtm

**Comment by danpopSD** [comment]
IMHO - one of the best engineers on the planet.   ❤️  you @n3wscott .  A BIG BIG +1 from POP

**Pull request merged**